src/fdtransfer.h

```diff
@@ -32,6 +32,7 @@ struct fd_request {
 struct perf_fd_request {
     struct fd_request header;
     int tid;
+    int target_cpu;
```

Comment:
Do we need to ensure backward compatibility with the protocol here?

Reply:
Good point! We don't need it to be backward compatible, though.